content/pt/docs/_index.md

```diff
@@ -3,23 +3,7 @@ title: Documentação
 linkTitle: Docs
 menu: { main: { weight: 10 } }
 htmltest:
-  IgnoreDirs:
-    # TODO drop next lines after https://github.com/open-telemetry/opentelemetry.io/issues/5555 is fixed for these pages:
-    - ^pt/docs/concepts/components/
-    - ^pt/docs/concepts/glossary/
-    - ^pt/docs/concepts/signals/baggage/
-    - ^pt/docs/languages/erlang/instrumentation/
-    - ^pt/docs/languages/erlang/sampling/
-    - ^pt/docs/languages/js/instrumentation/
-    - ^pt/docs/languages/js/sampling/
-    - ^pt/docs/languages/net/instrumentation/
-    - ^pt/docs/languages/net/libraries/
-    - ^pt/docs/languages/net/shim/
-    - ^pt/docs/languages/php/instrumentation/
-    - ^pt/docs/languages/python/instrumentation/
-    - ^pt/docs/languages/ruby/instrumentation/
-    - ^pt/docs/languages/ruby/sampling/
-    - ^pt/docs/zero-code/php/
+  IgnoreDirs: {}
```

Comment:
I missed this, and the CI checks didn't report it. FYI, I've removed this line, and fixed the CI checks via:

- #5612